docs/configuration/index.md

```diff
@@ -1846,6 +1846,7 @@ The Druid SQL server is configured through the following properties on the Broke
 |`druid.sql.planner.metadataSegmentPollPeriod`|How often to poll coordinator for published segments list if `druid.sql.planner.metadataSegmentCacheEnable` is set to true. Poll period is in milliseconds. |60000|
 |`druid.sql.planner.authorizeSystemTablesDirectly`|If true, Druid authorizes queries against any of the system schema tables (`sys` in SQL) as `SYSTEM_TABLE` resources which require `READ` access, in addition to permissions based content filtering.|false|
 |`druid.sql.planner.useNativeQueryExplain`|If true, `EXPLAIN PLAN FOR` will return the explain plan as a JSON representation of equivalent native query(s), else it will return the original version of explain plan generated by Calcite. It can be overridden per query with `useNativeQueryExplain` context key.|false|
+|`druid.sql.planner.maxNumericInFilters`|If set to a value between 1 and 10,000, Druid will allow numeric values specified for IN part of the query not to exceed this user defined values and queries with more than 10,000 numeric IN filters will not be run by Druid unless they are cast to String.|`10000`|
```

Comment:
Would be good if we specify that it could be overridden in the context using the `maxNumericInFilters` key.

Reply:
Good point. I'll add that it can be overridden through the query context

Reply:
Done